src/mysql-util/src/bin/mysql-gen.rs

```diff
@@ -0,0 +1,200 @@
+                FROM customers, products_on_hand
+                WHERE products_on_hand.quantity > 30
+                ORDER BY rand() LIMIT 1"#,],
+        vec![r#"INSERT INTO orders (order_date, purchaser, quantity, product_id)
+                SELECT date(now()), customers.id, 4, products_on_hand.product_id
+                FROM customers, products_on_hand
+                WHERE products_on_hand.quantity > 40
+                ORDER BY rand() LIMIT 1"#,],
+        vec![r#"INSERT INTO orders (order_date, purchaser, quantity, product_id)
+                SELECT date(now()), customers.id, 5, products_on_hand.product_id
+                FROM customers, products_on_hand
+                WHERE products_on_hand.quantity > 50
+                ORDER BY rand() LIMIT 1"#,],
+        vec![r#"INSERT INTO addresses (customer_id, street, city, state, zip, type)
+                VALUES (
+                   (SELECT id FROM customers ORDER BY rand() LIMIT 1),
+                   '1 Infinite View',
+                   'New York',
+                   'New York',
+                   '10001',
+                   'BILLING'
+                  )"#,],
+        vec![r#"INSERT INTO addresses (customer_id, street, city, state, zip, type)
+                VALUES (
+                   (SELECT id FROM customers ORDER BY rand() LIMIT 1),
+                   '1 Fantastic Sink',
+                   'New York',
+                   'New York',
+                   '10001',
+                   'SHIPPING'
+                  )"#,],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('alex', 'materialize', 'am@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('cameron', 'materialize', 'cm@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('frankie', 'materialize', 'fm@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('jaime', 'materialize', 'jm@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('kyle', 'materialize', 'km@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('mattie', 'materialize', 'mm@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('nicky', 'materialize', 'nm@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('pat', 'materialize', 'pm@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('quinn', 'materialize', 'qm@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('riley', 'materialize', 'rm@materialize.com')",],
+        vec!["INSERT IGNORE INTO customers (first_name, last_name, email) VALUES ('taylor', 'materialize', 'tm@materialize.com')",],
```

Comment:
no action, but recruiters crawling github for emails will have a field day (:

Reply:
I'll change them to be `.io` :)